src/app/governance/govhub/components/ToolsCardContent.tsx

```diff
@@ -0,0 +1,40 @@
+import { clsx } from 'clsx'
+
+import type { CTAProps } from '@/types/sharedProps/ctaType'
+
+import { Heading } from '@/components/Heading'
+import { TextLink } from '@/components/TextLink'
+
+type ToolsCardContentProps = {
+  title: string
+  description: string
+  listItems: Array<string>
+  cta: CTAProps
+}
+
+export function ToolsCardContent({
+  title,
+  description,
+  listItems,
+  cta,
+}: ToolsCardContentProps) {
+  return (
+    <div className="flex flex-col gap-6">
+      <div>
+        <Heading tag="h3" variant="lg">
+          {title}
+        </Heading>
+        <p className="mb-6 mt-3">{description}</p>
+        <Heading tag="h4" variant="md">
+          Purpose
+        </Heading>
```

Comment:
The spacing between purpose and the texts below seems smaller on Figma: https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=6695-92069&t=gwKMxyp7Br40sRQl-4

They also seem to use slightly different font weight? Not sure.

Reply:
Yes, it's true. I have an [open request pending on Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA?node-id=6621-139401#975778838). Let's go with what we have for now.